ckan/controllers/package.py

```diff
@@ -590,7 +590,8 @@ def resource_edit(self, id, resource_id, data=None, errors=None,
 
             context = {'model': model, 'session': model.Session,
                        'api_version': 3, 'for_edit': True,
-                       'user': c.user or c.author, 'auth_user_obj': c.userobj}
+                       'user': c.user or c.author, 'auth_user_obj': c.userobj,
+                       'action': 'edit'}
```

Comment:
But `'action'` didn't exist in the context dict before. Is there a reason you're adding it here and below?